eng/SourceBuild.props

```diff
@@ -1,3 +1,5 @@
+<!-- Whenever altering this or other Source Build files, please include @dotnet/source-build-internal as a reviewer. -->
```

Comment:
Instead of adding this comment, add lines with this alias for these files to 

https://github.com/dotnet/runtime/blob/main/.github/CODEOWNERS

It will ensure that you are automatically included on PRs that modify these.

Reply:
Initially wanted to add both the comment and the `CODEOWNERS` entry, but for a team to be added as a code owner it needs to have [write permissions](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) to the repo and I wanted to verify first if this is something we would need. I should've mentioned that in the description though, my bad, forgot to edit it.

If you are OK with granting write permissions to `dotnet/source-build-internal`, I will add the `CODEOWNERS` entry.

Reply:
Thank you, will do

Reply:
The e-mail thread unfortunately is taking a bit longer then expected.

Would it make sense from your perspective to merge the PR with the comments only and return to the subject of `CODEOWNERS` entry later? Thank you.